optuna/samplers/_cmaes.py

```diff
@@ -287,13 +314,7 @@ def infer_relative_search_space(
                 # `Trial`.
                 continue
 
-            if not isinstance(
-                distribution,
-                (
-                    optuna.distributions.FloatDistribution,
-                    optuna.distributions.IntDistribution,
-                ),
-            ):
+            if not isinstance(distribution, (FloatDistribution, IntDistribution)):
```

Comment:
Do you think we can leave a todo comment about the support of `CategoricalDistribution`, which contains only two choices?